server/api/routes/users.py

```diff
@@ -1,42 +1,67 @@
 from flask import Blueprint, request
 from flask_jwt_extended import create_access_token
 from database import User
+from api.routes.validators.users import valid_login, valid_registration
 
 users = Blueprint('users', __name__)
 
-
 @users.route('/login', methods=['POST'])
 def login():
     """POST users/login
 
     Desc: Logs in a user
+          Checks if any fields are blank
+          Checks if an account exists at all
+          Checks if an account exists and the password was incorrect
 
     Returns:
         200: success - true
-        401: success - false
+        401: password was incorrect, unauthorized access
+        404: no account was found or field was empty
     """
     req = request.get_json()
-    user = User.objects(email=req['email'], password=req['password']).first()
-    if user:
-        access_token = create_access_token(identity=user)
+    is_valid, errors = valid_login(req)
+    if not is_valid:
+        return errors, 404
+
+    user = User.objects(email=req['email']).first()
+    if not user:
         return {
-            'token': access_token
-        }, 200
-    else:
+            'error': 'No account with this email exists.'
```

Comment:
Break it down into the email and password fields here. It just makes it easier on the frontend team so that they don't have to figure out what field the error message is under. If every input field for every form always has an error message that's present in a field named after the input field, then it makes everything much more consistent